include/triton/core/tritonbackend.h

```diff
@@ -1569,6 +1569,60 @@ TRITONBACKEND_ISPEC TRITONSERVER_Error* TRITONBACKEND_ModelBatchInitialize(
 TRITONBACKEND_ISPEC TRITONSERVER_Error* TRITONBACKEND_ModelBatchFinalize(
     void* userp);
 
+/// Get all information about an output tensor by name.  The tensor data is
```

Comment:
You also need to increment the API version at the top of the file.

Reply:
Good catch! I must've lost this change transferring from my test branch.